### PR TITLE
GC-231: Add limited token login Facebook

### PR DESCRIPTION
**Pull request by nguyenok**

### CODE REVIEW

File: extensions/currency/migration/Version-1.0.2.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  await execute(
+    connection,
+    `ALTER TABLE currency
+    RENAME COLUMN currency TO code;`
+  );
+
+  await execute(
+    connection,
+    `ALTER TABLE currency
+    ADD COLUMN is_default int DEFAULT 0, ADD COLUMN name TEXT`
```
Comment (GohubUser) [inline extensions/currency/migration/Version-1.0.2.js:13]: Missing something @nguyenok ?

File: extensions/facebook_login/api/createFacebookCustomer/[bodyParser]createFacebookCustomer.js [hunk 2]
```diff
@@ -1,16 +1,61 @@
...
+const { getDefaultLanguage } = require('../../services/getDefaultLanguage');
+const { getDefaultCurrency } = require('../../services/getDefaultCurrency');
+const {
+  createLanguageResponse
+} = require('../../services/mapper/createLanguageResponse');
+const {
+  createCurrencyResponse
+} = require('../../services/mapper/createCurrencyResponse');
+
+const facebookUserExample = {
```
Comment (GohubUser) [inline extensions/facebook_login/api/createFacebookCustomer/[bodyParser]createFacebookCustomer.js:23]: Please move it to the env.

File: extensions/google_login/api/createGoogleCustomer/[bodyParser]createGoogleCustomer.js [hunk 3]
```diff
@@ -6,6 +6,25 @@ const {
...
+const { getDefaultLanguage } = require('../../services/getDefaultLanguage');
+const { getDefaultCurrency } = require('../../services/getDefaultCurrency');
+const {
+  createLanguageResponse
+} = require('../../services/mapper/createLanguageResponse');
+const {
+  createCurrencyResponse
+} = require('../../services/mapper/createCurrencyResponse');
+
+const facebookUserInfoExample = {
```
Comment (GohubUser) [inline extensions/google_login/api/createGoogleCustomer/[bodyParser]createGoogleCustomer.js:18]: Please remove it when you don't need @nguyenok

File: packages/evershop/src/modules/customer/migration/Version-1.0.4.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    `ALTER TABLE customer
+    DROP COLUMN full_name;`
+  );
+
+  await execute(
+    connection,
+    `ALTER TABLE customer
+    ADD COLUMN first_name VARCHAR(50),
+    ADD COLUMN last_name VARCHAR(50),
+    ADD COLUMN avatar_url TEXT`
```
Comment (GohubUser) [inline packages/evershop/src/modules/customer/migration/Version-1.0.4.js:17]: Do we need a semicolon?

File: packages/evershop/src/modules/translation/migration/Version-1.0.0.js [hunk 5]
```diff
@@ -40,8 +40,8 @@ module.exports = exports = async (connection) => {
   await execute(
     connection,
     `ALTER TABLE customer
-    ADD COLUMN language_code varchar(20),
-    ADD CONSTRAINT "FK_CUSTOMER_LANGUAGE" FOREIGN KEY ("language_code") REFERENCES "language" ("code") ON DELETE SET NULL;`
+    ADD COLUMN language_id uuid,
```
Comment (GohubUser) [inline packages/evershop/src/modules/translation/migration/Version-1.0.0.js:43]: I think you would check an existed column or create a new migration @nguyenok

File: extensions/facebook_login/api/createFacebookCustomer/[bodyParser]createFacebookCustomer.js (hunk 2 above)
Comment (nguyenok) [reply extensions/facebook_login/api/createFacebookCustomer/[bodyParser]createFacebookCustomer.js]: this line of code has been removed

File: packages/evershop/src/modules/customer/migration/Version-1.0.4.js (hunk 4 above)
Comment (nguyenok) [reply packages/evershop/src/modules/customer/migration/Version-1.0.4.js]: I added a semicolon to this

File: packages/evershop/src/modules/translation/migration/Version-1.0.0.js (hunk 5 above)
Comment (nguyenok) [reply packages/evershop/src/modules/translation/migration/Version-1.0.0.js]: I've created a new migration version for this @GohubUser

**Review by GohubUser — APPROVED** [review]
LGTM

**Pull request merged**